tools/AzureRM.BootStrapper/help/Get-AzureRmProfile.md

```diff
@@ -11,7 +11,7 @@ List the supported AzureRM profiles.
 ## SYNTAX
 
 ```
-Get-AzureRmProfile [-ListAvailable]
+Get-AzureRmProfile [-ListAvailable] [-Force]
```

Comment:
See the previous review.  Force is likely not thei correct parameter name in this case.